### PR TITLE
AD contract update

### PR DESCRIPTION
**Pull request by blimpa**
@boet1

### CODE REVIEW

File: Sky Atlas/Sky Atlas.html [hunk 1]
```diff
@@ -12215,8 +12215,8 @@ <h1>Active Data</h1>
             || UPMaker | <a
               href="https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd">https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd</a>
             <a href="https://etherscan.io/verifySig/16997">https://etherscan.io/verifySig/16997</a> | <a
-              href="https://etherscan.io/address/0xc9C17EE58810172622b43B09C9C9FF31e50f5179">https://etherscan.io/address/0xc9C17EE58810172622b43B09C9C9FF31e50f5179</a>
-            <a href="https://etherscan.io/verifySig/38947">https://etherscan.io/verifySig/38947</a> | <a
+              href="https://etherscan.io/address/0xe0e9882012ecbcfec5e1fa04102b07185dfdeab7">https://etherscan.io/address/0xe0e9882012ecbcfec5e1fa04102b07185dfdeab7</a>
+            <a href="https://etherscan.io/verifySig/259655">https://etherscan.io/verifySig/259655</a> | <a
```
Comment (boet1) [inline Sky Atlas/Sky Atlas.html:12219]:
```suggestion
            <a href="https://etherscan.io/verifySig/259652">https://etherscan.io/verifySig/259652</a> | <a
```
I think the correct signature is this one. The signature you added belongs to EA.

File: Sky Atlas/Sky Atlas.html [hunk 2]
```diff
@@ -12227,8 +12227,8 @@ <h1>Active Data</h1>
             || BLUE | <a
               href="https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf">https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf</a>
             <a href="https://etherscan.io/verifySig/19057">https://etherscan.io/verifySig/19057</a> | <a
-              href="https://etherscan.io/address/0xb18680092734394295d0591bb42f2bd3c184517e">https://etherscan.io/address/0xb18680092734394295d0591bb42f2bd3c184517e</a>
-            <a href="https://etherscan.io/verifySig/25260">https://etherscan.io/verifySig/25260</a> | <a
+              href="https://etherscan.io/address/0x73056E27D9cc08f963d41664C705392f305cFE7d">https://etherscan.io/address/0x73056E27D9cc08f963d41664C705392f305cFE7d</a>
```
Comment (boet1) [inline Sky Atlas/Sky Atlas.html:12230]:
```suggestion
              href="https://etherscan.io/address/0xe1FF7ECf038900dd14A1E7E3a8CCF23f71f6A1c2">https://etherscan.io/address/0xe1FF7ECf038900dd14A1E7E3a8CCF23f71f6A1c2</a>
```
It seems the delegate made a mistake and didn’t enter the correct address for the contract.

File: Sky Atlas/Sky Atlas.html [hunk 3]
```diff
@@ -12299,8 +12299,8 @@ <h1>Active Data</h1>
             || JuliaChang | <a
               href="https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7">https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7</a>
             <a href="https://etherscan.io/verifySig/34622">https://etherscan.io/verifySig/34622</a> | <a
-              href="https://etherscan.io/address/0x7938b304F1c28e85b6A021251C8CeefF20370f38">https://etherscan.io/address/0x7938b304F1c28e85b6A021251C8CeefF20370f38</a>
-            <a href="https://etherscan.io/verifySig/34620">https://etherscan.io/verifySig/34620</a> | <a
+              href="https://etherscan.io/address/0xd0B44469d0f230FC67d2e6d2e8D239699eD76Bb5">https://etherscan.io/address/0xd0B44469d0f230FC67d2e6d2e8D239699eD76Bb5</a>
```
Comment (boet1) [inline Sky Atlas/Sky Atlas.html:12302]:
```suggestion
              href="https://etherscan.io/address/0x5D4a96aFadb5505a8fF21ddeCa2Baf3A63301396">https://etherscan.io/address/0x5D4a96aFadb5505a8fF21ddeCa2Baf3A63301396</a>
```
It seems the same thing happened here, the delegate made a mistake and entered the address used to create the contract.

File: Sky Atlas/Sky Atlas.html [hunk 4]
```diff
@@ -12185,8 +12185,8 @@ <h1>Active Data</h1>
             | --------------------------------------------------------------------------------- || vigilant | <a
               href="https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61">https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61</a>
             <a href="https://etherscan.io/verifySig/16682">https://etherscan.io/verifySig/16682</a> | <a
-              href="https://etherscan.io/address/0xDE08aEf2b221274231b3547491ec8f0Fc80917e1">https://etherscan.io/address/0xDE08aEf2b221274231b3547491ec8f0Fc80917e1</a>
-            <a href="https://etherscan.io/verifySig/255532">https://etherscan.io/verifySig/255532</a> | <a
+              href="https://etherscan.io/address/0x1F68D75A0C7c2Af84Ef48d079A95cb5A0f413347">https://etherscan.io/address/0xDE08aEf2b221274231b3547491ec8f0Fc80917e1</a>
```
Comment (VoteWizard) [inline Sky Atlas/Sky Atlas.html:12188]:
```suggestion
              href="https://etherscan.io/address/0x1F68D75A0C7c2Af84Ef48d079A95cb5A0f413347">https://etherscan.io/address/0x1F68D75A0C7c2Af84Ef48d079A95cb5A0f413347</a>
```

**Review by VoteWizard — DISMISSED** [review]
Approved, thanks @blimpa

**Review by MakerJanSky — APPROVED** [review]
Nice work, all.

**Review by VoteWizard — APPROVED** [review]
Approved

**Pull request merged**